src/content/structured/styles/content-style.mdx

```diff
@@ -34,7 +34,7 @@ The way we write, and [how we present it](/styles), can make or break an experie
 
 This guide applies to content you write in apps (like error messages) through to user guides and corporate comms.
 
-Our goal is to create informative, helpful and clear content. This isn't about 'dumbing content down' - our job is to 'open it up'. We want as many people as possible to understand and engage with what we write.
+Our goal is to create informative, helpful and clear content. This isn't about 'dumbing content down'—our job is to 'open it up'. We want as many people as possible to understand and engage with what we write.
```

Comment:
```suggestion
Our goal is to create informative, helpful and clear content. This isn't about 'dumbing content down' — our job is to 'open it up'. We want as many people as possible to understand and engage with what we write.
```

Reply:
This should be an em rule (&emdash;), I'm just not sure on how to format that yet but hoping a dev can help. Em rules usually have no spaces around them and are used for emphasis when marking a break in a sentence. Alternatively, it could be an en rule with spaces around it (not a hyphen), but I think an em rule works best here :)